### ISSUE TITLE
This mod is still alive?

### ISSUE THREAD

**Issue by Xen1290** [BuiltBrokenModding/AtomicScience#1]
@DarkGuardsman  @dmodoomsirius This mod is still alive? Now EDX(alias Resonant Induction/Atomic Science) is no longer updated by calclavia, and currently there are no known working versions for minecraft 1.7.10. Will you update this mod? A lot of people miss that mod.(Fusion Reactor/Fission Reactor/PARTICLE ACCELERATOR...
And sorry for possible english mistakes.

**Comment by dmodoomsirius** [comment]
Yes we will be updating this mod. We are working on it along with mad science. We can't give a eta on the release.

**Comment by Xen1290** [comment]
Thanks!

**Comment by Xen1290** [comment]
Thanks! I hope this awesome mod will be released in the next weeks, and good work!

**Comment by Xen1290** [comment]
Thanks! I hope this awesome mod will be released in the next weeks, and good development!

**Comment by DarkGuardsman** [comment]
Its going to be a while before release of the mod.

**Comment by Xen1290** [comment]
Hello? Any news for the possibile 1.7.10/1.8.9 update?

**Comment by DarkGuardsman** [comment]
"Its going to be a while before release of the mod"

Meaning its going to be a while, a few months until we can even get started. The plan is to finish the industry mod we are working on. Then make atomic science an addon for that industry mod. This way the content matches very well. Including better support for varies features, etc. Problem is the industry mod will take a while. Especially since I'm about the only one working on it at the moment. The other two developers are busy with life.

**Comment by Xen1290** [comment]
Ah, now i understand. I thought that atomic science was a standalone mod. Thanks for the explaination. Sorry about that.

**Comment by DarkGuardsman** [comment]
np, as much as I would like to make it standalone the quality will be higher as an addon

**Comment by spacebuilder2020** [comment]
Any plans to bring atomic science back in the near future?
Sorry about bumping an old thread, but I have been wanting to play atomic science on 1.7.10 for quite a while.

**Comment by DarkGuardsman** [comment]
Yes, it's just not a high priority. As one of the few devs for BBM I have a lot of projects to maintain on top of handling other tasks. Sadly this means Atomic Science has been untouched for a long while. I'm still not sure when I will be able to get to working on the project.

**Comment by spacebuilder2020** [comment]
Is there any 1.6.4 source code available? I am willing to take a shot at updating it.

**Comment by DarkGuardsman** [comment]
Yes, as well there are 2 others that are working on unofficial updates. What I can do here later is upload the code for people to contribute towards the official update.

**Comment by spacebuilder2020** [comment]
Thanks

**Comment by spacebuilder2020** [comment]
Where is the 1.6.4 source located?  I was able to find the binaries but not the source?

**Comment by DarkGuardsman** [comment]
Sorry have yet to upload it, will do now

**Comment by DarkGuardsman** [comment]
I seem to have misplaced my copy of the code, I will keep looking for it.

@dmodoomsirius if you have a copy could you upload it here https://github.com/BuiltBrokenModding/AtomicScience-Classic or give me a link to the zip so I can upload it

**Comment by spacebuilder2020** [comment]
ok

**Comment by spacebuilder2020** [comment]
@DarkGuardsman @dmodoomsirius Have either of you managed to find a copy of the source code for atomic science?

**Comment by DarkGuardsman** [comment]
@spacebuilder2020 I'm waiting on @dmodoomsirius who is is currently busy with work and build servers. I'll see if I can ask someone else if they have a src copy. As I can not locate mine and to get a copy requires manually reading through Resonant Induction or EDX commits. Which is how we got the copy in the first place. If we can not find a copy I might end up reading through the commits to find the source. However, this has to wait until after my current work load.

**Comment by spacebuilder2020** [comment]
ok. no problem

Take your time.

**Comment by DarkGuardsman** [comment]
@Hennamann Gave me a hand and we have at least an orginal copy of the Resonant Induction code https://github.com/BuiltBrokenModding/AtomicScience-Classic

I will continue to find the original code (with git history) but this should get us started.

**Comment by spacebuilder2020** [comment]
lol... Just a few minutes ago I found the commit with the source...
